pom.xml

```diff
@@ -3,7 +3,9 @@
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
-    <version>1.509.1</version><!-- which version of Jenkins is this plugin built against? -->
+    <version>1.580.2</version><!-- which version of Jenkins is this plugin built against? (Current LTS as at 14 Jan 2015)-->
```

Comment:
Fair enough, I had updated it because it did resolve an issue when building with Java 8, i.e. you got the following error:

    Tests in error:
          org.jvnet.hudson.test.JellyTestSuiteBuilder$JellyTestSuite(org.jvnet.hudson.test.junit.FailedTest): -1
    testCliSanity(org.jvnet.hudson.test.PluginAutomaticTestBuilder$CliSanityTest): second instance

however as you correctly point out, this is not critical and will work with JDK 7 so should not be changed, I shall revert this